app/assets/stylesheets/application.css

```diff
@@ -14,4 +14,33 @@
  *= require_self
  *= require font-awesome
  *= paginator.css
- */
\ No newline at end of file
+ */
+
+html,
+body {
+  height: 100%;
+}
+
+
+.btn {
+    background-color: ghostwhite;
+    border-radius: 0.5rem;
+    color: black;
+    cursor: pointer;
+    font-size: 0.75rem;
+    padding: 0.4rem 0.5rem;
+}
+
+.btn--gray {
+    background-color: #64758B;
```

Comment:
So the number one reason to use tailwind in my opinion is the constraints. But here we are not using them. With constraints we would never allow users to choose `0.4rem and 0.5rem` instead we would use `py-2 px-3` or something that follows our scale.

We should never be hard coding any colours into our styles. If we are defining new colours we should use CSS variables for that. We can also extend tailwind with new colours, or replace existing ones. 

Here I would expect something like

```
.btn {
  @apply bg-zinc-50;
}
```

So that it does not go outside of the design system.